### PR TITLE
blog: june tilt commit of the month

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: blog/_posts/2019-06-27-commit-of-the-month-june.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+keywords:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+---
+
+Welcome to a new series of blog posts we're calling Tilt Commit of the Month. Commit of the Month is a lightweight way to highlight work that goes on in the Tilt project that might fly under the radar otherwise.
+
+We’re already cheating: this month we’re going to highlight a bunch of commits related to better detecting and working better with different kinds of Kubernetes clusters.
```
Comment (maiamcc) [inline blog/_posts/2019-06-27-commit-of-the-month-june.md]: the cheating bit is that it's MULTIPLE commits, right? that doesn't land -- consider rewording, using italics, etc.

File: blog/_posts/2019-06-27-commit-of-the-month-june.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* [k8s: auto-detect microk8s registry](https://github.com/windmilleng/tilt/commit/d293a0ba0216711c855526100490e21733ada194)
+* [docker: dynamically switch between local docker daemon and the in-cluster daemon](https://github.com/windmilleng/tilt/commit/af5a0a7e0c32c8b55117c9f22c28008f8af67d2d)
+* [engine: fix exec syncing on busybox](https://github.com/windmilleng/tilt/commit/6678de099fd559b8e02240a3a807d3b06e65aff8)
+
+
+## What they do
+Combined these changes make Tilt better support KIND, microk8s and minikube. On microk8s we now detect whether we should use a registry built in microk8s, or an external one. Finally [`live_update`](https://docs.tilt.dev/live_update_reference.html) on KIND clusters no longer results in full rebuilds.
+
+## Why they're important
+Aside from making Tilt work better on various Kubernetes clusters for users these change also pave the way for us to run our integration tests across many different types of Kubernetes clusters, rather than just one.
```
Comment (maiamcc) [inline blog/_posts/2019-06-27-commit-of-the-month-june.md]:
```suggestion
Aside from making Tilt work better on various Kubernetes clusters, these change also pave the way for us to run our integration tests across many different types of Kubernetes clusters, rather than just one.
```

Also maybe needs a "so what" -- "we'll be better able to catch edge-cases that affect only certain clusters... that means fewer breakages/rollbacks/bug fixes..."

File: blog/_posts/2019-06-27-commit-of-the-month-june.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+We’re already cheating: this month we’re going to highlight a bunch of commits related to better detecting and working better with different kinds of Kubernetes clusters.
+
+Commits:
+* [k8s: auto-detect microk8s registry](https://github.com/windmilleng/tilt/commit/d293a0ba0216711c855526100490e21733ada194)
+* [docker: dynamically switch between local docker daemon and the in-cluster daemon](https://github.com/windmilleng/tilt/commit/af5a0a7e0c32c8b55117c9f22c28008f8af67d2d)
+* [engine: fix exec syncing on busybox](https://github.com/windmilleng/tilt/commit/6678de099fd559b8e02240a3a807d3b06e65aff8)
+
+
+## What they do
+Combined these changes make Tilt better support KIND, microk8s and minikube. On microk8s we now detect whether we should use a registry built in microk8s, or an external one. Finally [`live_update`](https://docs.tilt.dev/live_update_reference.html) on KIND clusters no longer results in full rebuilds.
```
Comment (maiamcc) [inline blog/_posts/2019-06-27-commit-of-the-month-june.md]:
```suggestion
Combined, these changes improve Tilt support for KIND, microk8s, and minikube. On microk8s we now detect whether we should use a registry built in microk8s, or an external one. Finally [`live_update`](https://docs.tilt.dev/live_update_reference.html) on KIND clusters no longer results in full rebuilds.
```

Comment (maiamcc) [reply blog/_posts/2019-06-27-commit-of-the-month-june.md]: you mention 3 commits, and only list two things that they do. (i.e. "finally..." feels like it should come after TWO things but it only comes after one.)

Comment (maiamcc) [reply blog/_posts/2019-06-27-commit-of-the-month-june.md]: ALSO: consider hyperlinking the diff clusters mentioned here in case ppl are unfamiliar

File: blog/_posts/2019-06-27-commit-of-the-month-june.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+  - commit
+keywords:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
```
Comment (maiamcc) [inline blog/_posts/2019-06-27-commit-of-the-month-june.md]: Should we have a "Commit of the Month" (/ "CotM") tag so that users can filter by that?

File: blog/_posts/2019-06-27-commit-of-the-month-june.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+slug: june-tilt-commit-of-the-month
+date: 2019-07-03T15:26:34.743Z
+author: dmiller
+layout: blog
+title: "Tilt Commit of the Month"
```
Comment (maiamcc) [inline blog/_posts/2019-06-27-commit-of-the-month-june.md]: consider a title (or subtitle) with a bit more context, e.g. "Commit of the Month: Support More k8s Clusters"

File: blog/_posts/2019-06-27-commit-of-the-month-june.md (hunk 4 above)
Comment (jazzdan) [reply blog/_posts/2019-06-27-commit-of-the-month-june.md]: FWIW we don't seem to expose these tags to users?
